scvi/external/solo/_model.py

```diff
@@ -324,15 +324,18 @@ def train(
         return runner()
 
     @torch.no_grad()
-    def predict(self, soft: bool = True) -> pd.DataFrame:
+    def predict(
+        self, soft: bool = True, include_simulated_doublets: bool = True
```

Comment:
```suggestion
        self, soft: bool = True, include_simulated_doublets: bool = False
```
Default should be false right? I don't think most users care to see the sim doubs?

Reply:
yup!